README.md

```diff
@@ -110,15 +110,50 @@ namespace FileUploader
 ```
 
 ## Running Minio Client Examples
-* Clone this repository.
+#### On Windows 
+* Clone this repository and open the Minio.Sln in Visual Studio 2017.
 
-* Build the project to produce the Minio.Examples console app.
+* Enter your credentials and bucket name, object name etc.in Minio.Examples/Program.cs
+  Uncomment the example test cases such as below in Program.cs to run an example.
+```cs
+  //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
+```
+* Run the Minio.Client.Examples.NET452 or Minio.Client.Examples.NetCore project from Visual Studio
+#### On Linux (Ubuntu 16.04 and above)
+
+##### Setting up Mono and .NETCore on Linux
+<blockquote> NOTE: minio-dotnet requires mono 5.0.1 stable release and .NET Core 1.0 SDK to build on Linux. </blockquote>
+
+* Install .NETCore and Mono by running [mono_install.sh](https://github.com/minio/minio-dotnet/blob/master/mono_install.sh)
+
+```
+$ ./mono_install.sh    
+```
+##### Running Minio.Examples 
+* Clone this project
+
+```
+$ git clone https://github.com/minio/minio-dotnet && cd minio-dotnet 
+```
 
-* Move into Minio.Examples directory and enter your credentials and bucket name, object name etc.
+* Enter your credentials and bucket name, object name etc. in Minio.Examples/Program.cs
   Uncomment the example test cases such as below in Program.cs to run an example.
 ```cs
   //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
 ```
+* To run .NET4.5.2 example,
+```
+$ msbuild /p:Configuration=.net4.5.2 /t:Clean 
+$ msbuild /p:Configuration=.net4.5.2
+$ ./Minio.Examples/Minio.Client.Examples.Net452/bin/Debug/Minio.Client.Examples.Net452.exe 
```

Comment:
This path is incorrect after running the previous command 

```
Skipping target "GenerateTargetFrameworkMonikerAttribute" because all output files are up-to-date with respect to the input files.
/usr/lib/mono/msbuild/15.0/bin/Microsoft.CSharp.CurrentVersion.targets(141,9): warning MSB3884: Could not find rule set file "MinimumRecommendedRu
les.ruleset". [/root/minio-dotnet/Minio.Examples/Minio.Client.Examples.Net452/Minio.Client.Examples.Net452.csproj]
CoreCompile:
Skipping target "CoreCompile" because all output files are up-to-date with respect to the input files.
_CopyAppConfigFile:
  Copying file from "obj/.net4.5.2/Minio.Client.Examples.Net452.csproj.Minio.Client.Examples.Net452.exe.config" to "bin/.net4.5.2/Minio.Client.Exa
mples.Net452.exe.config".
CopyFilesToOutputDirectory:
  Minio.Client.Examples.Net452 -> /root/minio-dotnet/Minio.Examples/Minio.Client.Examples.Net452/bin/.net4.5.2/Minio.Client.Examples.Net452.exe
Done Building Project "/root/minio-dotnet/Minio.Examples/Minio.Client.Examples.Net452/Minio.Client.Examples.Net452.csproj" (default targets).
Done Building Project "/root/minio-dotnet/Minio.sln" (default targets).
```

The path is not inside `Debug/`